api/test/e2e/h3-data/h3-impact-map.spec.ts

```diff
@@ -41,7 +41,7 @@ describe('H3 Data Module (e2e) - Impact map', () => {
   afterAll(async () => {
     await dropH3DataMock([fakeTable]);
     await deleteImpactMapMockData();
-    return app.close();
+    await app.close();
```

Comment:
these are functionally equivalent AFAIK. could you explain the change please?

Reply:
Because we are awaiting to app.close() in 90% of the tests suites, I think it's cleaner to be consistent.

Reply:
we should probably prefer the return version, but that's outside of scope here. leave it as you have it :)